src/System.Reflection.MetadataLoadContext/src/System/Reflection/TypeLoading/General/Sentinels.cs

```diff
@@ -33,7 +33,8 @@ private sealed class SentinelMethod : RoMethod
             public sealed override IEnumerable<CustomAttributeData> CustomAttributes => throw null;
             public sealed override bool IsConstructedGenericMethod => throw null;
             public sealed override bool IsGenericMethodDefinition => throw null;
-            public sealed override bool Equals(object obj) => throw null;
+            // Called by MethodInfo operator==, behave as null
+            public sealed override bool Equals(object obj) => obj is null ? true : false;
```

Comment:
Sheesh, more `true : false` crap. I shouldn't have mentioned that workaround in that issue.

Reply:
:)

Can just return `obj is null`

Reply:
:)

Can just return obj is null; is reflex